src/cli.js

```diff
@@ -196,9 +196,19 @@ function loadIgnores(params) {
   if (!file && !params.exclude) {
     return [];
   }
+  if (file)
+    file = fs.realpathSync(file);
 
   var lines = (file ? shjs.cat(file) : "").split("\n");
-  lines.unshift(params.exclude || "");
+  if (params.exclude && path.resolve(params.exclude) ===
```

Comment:
this could be simplified by assigning exclude e.g.
```
var exclude = params.exclude || "";
if (exclude && path.isAbsolute(exclude)) {
  exclude = ...
}
lines.unshift(exclude);
```